index.html

```diff
@@ -771,8 +769,8 @@ <h2>Extensions to the `Navigator` interface</h2>
         </div>
     </section>
     <section>
-        <h1>Declaring candidate regions for direct manipulation behaviors</h1>
-        <p>As noted in <a href="#attributes-and-default-actions">Attributes and Default Actions</a>, viewport manipulations (panning and zooming) cannot be suppressed by canceling a pointer event. Instead, authors must explicitly define which of these behaviors they want to allow, and which they want to suppress, using the <code>touch-action</code> CSS property.</p>
+        <h1><dfn data-lt="direct manipulation behavior">Declaring candidate regions for direct manipulation behaviors</dfn></h1>
```

Comment:
Optional nit: I am a bit inclined towards having the word "declarative" in the anchor label, because that the main behavior diff here vs, say, Touch Events.  What about "declarative direct manipulation" or +" behavior"?

Reply:
which i see is what i wanted to change this to originall,y but seem not to have done so in the end...oops

Reply:
+1 for "declaring ..." (pun intended :smile:)